mesh/pumi.hpp

```diff
@@ -57,8 +56,6 @@ class PumiMesh : public Mesh
    PumiMesh(apf::Mesh2* apf_mesh, int generate_edges = 0, int refine = 1,
             bool fix_orientation = true);
 
-   using Mesh::Load;
```

Comment:
Clang gives an "overloaded virtual function" warning without this

Reply:
Hi @psocratis 

Thanks for catching these. The two `getPumiNodeXis` routines are outdated and I will remove them shortly.

I am not able to get the clang warning on my build. Do you mind sharing your build options? I guess the one that I am looking for is the extra cxx flags that you are using. I guess using the MFEM build that would be specified by providing `CXXFLAGS` to `make config`

Reply:
I am using the default debug flags for mfem. 
make pdebug -j

Reply:
I think it's just `-g -Wall`

Reply:
got it thanks.

Reply:
These are fixed in the following two commits
https://github.com/mortezah/mfem/commit/78a8c79755344e40a85c0228c3744169ff9bd16a
https://github.com/mortezah/mfem/commit/40fb14767668b793de74b5f4b12025da26f12e2c